tests/functional/Ticket.php

```diff
@@ -560,6 +560,144 @@ public function testUpdateTicketWithActors(array $actors_input, array $expected_
         $this->checkActors($ticket, $expected_actors);
     }
 
+    /**
+     * Test that ticket anonymous users notifications are successfully updated.
+     *
+     * This method tests the functionality of updating ticket notifications for anonymous users.
+     * It creates a ticket with different actors (requester, observer, assign) and enables notifications.
+     * It then adds a follow-up to the ticket to trigger notifications and checks the notification queue.
+     * Finally, it updates the anonymous users to disable their notifications and checks the notification queue again.
+     *
+     * @return void
+     */
+    public function testUpdateActorsNotification(): void
+    {
+        /** @var \DBmysql $DB */
+        global $DB, $CFG_GLPI;
+
+        $this->login();
+
+        // Create ticket
+        $ticket_id = $this->createItem(
+            \Ticket::class,
+            [
+                'name'        => 'ticket title',
+                'content'     => 'a description',
+                'entities_id' => getItemByTypeName('Entity', '_test_root_entity', true),
+            ]
+        )->fields['id'];
```

Comment:
As explained IRL, you have to check each step of the test (afterwards too):
```suggestion
        // Create ticket
        $ticket_id = $this->createItem(
            \Ticket::class,
            [
                'name'        => 'ticket title',
                'content'     => 'a description',
                'entities_id' => getItemByTypeName('Entity', '_test_root_entity', true),
            ]
        )->fields['id'];
        $this->integer($ticket_id)->isGreaterThan(0);
```

Reply:
Fixed in commit https://github.com/glpi-project/glpi/pull/16070/commits/f474aae7213b52af6e5b7e9c66b3de582ba36cd0
Thank :+1:

Reply:
`$this->createItem()` already checks that item is correctly created.

Reply:
Thanks :+1: useless code has been removed https://github.com/glpi-project/glpi/pull/16070/commits/4e27a0d7970f917bef330706c616f0f495c05622